tools/ci_build/github/apple/objectivec/onnxruntime-mobile-objc.podspec.template

```diff
@@ -0,0 +1,52 @@
+Pod::Spec.new do |s|
+  s.name             = 'onnxruntime-mobile-objc'
+  s.version          = '@VERSION@'
+  s.summary          = 'ONNX Runtime Objective-C Pod'
+
+  s.description      = <<-DESC
+    Preview pod for the ONNX Runtime Objective-C API.
+                       DESC
+
+  s.homepage         = 'https://github.com/microsoft/onnxruntime'
+  s.license          = { :type => 'MIT', :file => @LICENSE_FILE@ }
+  s.author           = { "ONNX Runtime" => "onnxruntime@microsoft.com" }
+  s.source           = { :http => "@SOURCE_HTTP@" }
+  s.ios.deployment_target = '11.0'
+  s.preserve_paths = [ @LICENSE_FILE@ ]
+  s.default_subspec = "Core"
+
+  s.subspec "Core" do |core|
+    core.dependency "onnxruntime-mobile-c", "#{s.version}"
```

Comment:
If you are depend on the same version, maybe you can extract that our of the VERSION_NUMBER file, instead of supply it as a param to the scrip?
Not sure how it works at our current setup, ort-mobile-c ~> 1.8.0 ort-mobile-objc ~> 1.8.0-experimental